pkg/apis/core/v1/helper/helpers_test.go

```diff
@@ -1509,3 +1509,93 @@ func TestHugePageUnitSizeFromByteSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLoadBalancerStatusEqual(t *testing.T) {
+
+	testCases := []struct {
```

Comment:
It is better to put `name` for each test pattern and output the name at line 1597 when the pattern is failed.

Reply:
https://github.com/kubernetes/kubernetes/pull/96353